docs/_source/tutorials/notebooks/labelling_textclassification_setfit_zeroshot.ipynb

```diff
@@ -0,0 +1,645 @@
+        "\n",
+        "\n",
+        "Let's set up and train our zero-shot SetFit model. Please note that SetFit's approach to zero-shot is to create a synthetic dataset of training examples, which is different from other approaches (e.g., transformers zero-shot pipelines) where \"templated\" examples with label names are used at inference time."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "7LVpVjNtheHG"
+      },
+      "outputs": [],
+      "source": [
+        "labels = [\"change details\", \"card\", \"atm\", \"top up\", \"balance\", \"transfer\", \"exchange rate\", \"pin\"]\n",
+        "train_dataset = get_templated_dataset(\n",
+        "    candidate_labels=labels, \n",
+        "    sample_size=8, \n",
+        "    template=\"The customer request is about {}\"\n",
+        ")\n",
+        "\n",
+        "model = SetFitModel.from_pretrained(\"all-MiniLM-L6-v2\")\n",
+        "trainer = SetFitTrainer(\n",
+        "    model=model,\n",
+        "    train_dataset=train_dataset\n",
+        ")\n",
+        "trainer.train()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Sp3oSMi9snkp"
+      },
+      "source": [
+        "We can use our trained zero-shot model to predict over the dataset. We'll later load these predictions into Argilla and use them to speed-up the labelling process."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "VG7IUE4nzn41"
+      },
+      "outputs": [],
+      "source": [
+        "def get_predictions(texts):\n",
+        "    probas = model.predict_proba(texts, as_numpy=True)\n",
+        "    for pred in probas:\n",
+        "        yield [{\"label\": label, \"score\": score} for label, score in zip(labels, pred)]\n",
+        "\n",
+        "dataset = dataset.map(lambda batch: {\"prediction\": list(get_predictions(batch[\"text\"]))}, batched=True)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-ZQMFaRLXywT"
+      },
+      "source": [
+        "\n",
+        "Let's upload our dataset with vectors and predictions."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Kv2qsk_1XywT"
+      },
+      "outputs": [],
+      "source": [
+        "rg_ds = rg.DatasetForTextClassification.from_datasets(dataset)\n",
+        "\n",
+        "rg.log(\n",
+        "    name=\"banking77-topics-setfit\",\n",
+        "    records=rg_ds,\n",
+        "    chunk_size=50,\n",
+        ")"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Y-e_l9dtXywV"
+      },
+      "source": [
+        "## 🏷 Bulk labelling with the `find similar` and zero-shot predictions\n",
```

Comment:
```suggestion
        "## 🏷 Bulk labelling with `find similar` and zero-shot predictions\n",
```
I think this is slightly better? Not 100% on this one. The original isn't too bad either way.